src/project/cli-cmds/cmd-runner.ts

```diff
@@ -6,6 +6,10 @@ export enum BALLERINA_COMMANDS {
     TEST = "test", BUILD = "build", FORMAT = "format", RUN = "run", DOC = "doc"
 }
 
+export enum PROJECT_TYPE {
+    SINGLE_FILE = "SINGLE_FILE_PROJECT"
```

Comment:
Shall we add the `BUILD_PROJECT` and `BALR_PROJECT` kind as well? Yes, it is not used at the moment, but it would be better if we can add them and keep it synced with the compiler output as well.

Reply:
Updated.